### PR TITLE
feat(anime): updates to CFs

### PR DESCRIPTION
**Pull request by FonduemangVI**
# Pull Request

## Purpose

Updating various CFs to fix issues

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: docs/json/radarr/cf/anime-bd-tier-06-fansubs.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
       "negate": false,
       "required": false,
       "fields": {
-        "value": "\\b(ANE)\\b"
+        "value": "\\[ANE\\]"
```
Comment (nuxencs) [inline docs/json/radarr/cf/anime-bd-tier-06-fansubs.json:60]: would it make sense to use `-ANE$` as well?

File: docs/json/sonarr/cf/anime-bd-tier-06-fansubs.json [hunk 2]
```diff
@@ -66,7 +66,7 @@
       "negate": false,
       "required": false,
       "fields": {
-        "value": "\\b(ANE)\\b"
+        "value": "\\[ANE\\]"
```
Comment (nuxencs) [inline docs/json/sonarr/cf/anime-bd-tier-06-fansubs.json:69]: would it make sense to use `-ANE$` as well?

File: docs/json/radarr/cf/anime-bd-tier-06-fansubs.json (hunk 1 above)
Comment (FonduemangVI) [reply docs/json/radarr/cf/anime-bd-tier-06-fansubs.json:60]: Probably yes

File: docs/json/sonarr/cf/anime-bd-tier-06-fansubs.json (hunk 2 above)
Comment (FonduemangVI) [reply docs/json/sonarr/cf/anime-bd-tier-06-fansubs.json:69]: Probably yes

**Review by nuxencs — APPROVED** [review]
lgtm

**Pull request merged**